sonar-scanner-engine/src/main/java/org/sonar/scanner/scan/ProjectScanContainer.java

```diff
@@ -232,7 +234,13 @@ protected void doAfterStart() {
     }
     String branch = tree.root().definition().getBranch();
     if (branch != null) {
-      LOG.info("Branch key: {}", branch);
+      LOG.info("Branch key (deprecated): {}", branch);
+    }
+
+    String branchName = props.property(ScannerProperties.BRANCH_NAME);
+    if (branchName != null) {
+      BranchConfiguration branchConfig = getComponentByType(BranchConfiguration.class);
+      LOG.info("Branch name: {}, type: {}", branchName, branchConfig.branchType().toString().toLowerCase());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define the locale to be used in this String operation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1449)